brands/shop/books.json

```diff
@@ -29,6 +29,16 @@
       "shop": "books"
     }
   },
+  "shop/books|Bertrand": {
+    "countryCodes": ["pt"],
+    "tags": {
+      "brand": "Bertrand",
+      "brand:wikidata": "Q1866547",
+      "brand:wikipedia": "en: Livraria Bertrand",
```

Comment:
The space after en: might be an issue. I really don't know.

Reply:
Oops, good catch.